docs/dxp/7.x/en/developing-applications/reference/portlet-descriptor-to-osgi-service-property-map.md

```diff
@@ -107,7 +107,7 @@ The standard portlet descriptor mappings are first.
 |`/liferay-portlet-app/portlet/header-request-attribute-prefix`|`com.liferay.portlet.header-request-attribute-prefix=<String>` [7](#seven)|
 |`/liferay-portlet-app/portlet/header-timeout`|`header-timeout=<int>`|
 |`/liferay-portlet-app/portlet/icon`|`com.liferay.portlet.icon=<String>`|
-|`/liferay-portlet-app/portlet/include`|not supported|
+|`/liferay-portlet-app/portlet/include`|`com.liferay.portlet.include=<boolean>`|
```

Comment:
See https://github.com/liferay/liferay-portal/blob/bc74ba288a5f79488f5a72890162b742cee9d8bc/modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-tracker/src/main/java/com/liferay/portal/osgi/web/portlet/tracker/internal/PortletTracker.java#L696

This property affects the value of `com.liferay.portal.kernel.model.Portlet.isInclude()` which is used in many places.